src/assets/Standards/CollectibleStandards/ERC721/ERC721Standard.ts

```diff
@@ -173,4 +199,40 @@ export class ERC721Standard {
       );
     });
   };
+
+  /**
+   * Query if a contract implements an interface.
+   *
+   * @param address - Asset contract address.
+   * @param tokenId - tokenId of a given token in the contract.
+   * @returns Promise resolving an object containing the standard, tokenURI, symbol and name of the given contract/tokenId pair.
+   */
+  getDetails = async (address: string, tokenId?: string) => {
```

Comment:
What do you think about providing a return type for this?

Reply:
done here: https://github.com/MetaMask/controllers/pull/667/commits/8effcafddb6ca32d9566050ba50db569ceb00f07